src/libasr/pass/function_call_in_declaration.cpp

```diff
@@ -0,0 +1,354 @@
+#include <libasr/asr.h>
```

Comment:
Assuming this is copied from latest lfortran main.

Reply:
Yes.